src/stravalib/strava_model.py

```diff
@@ -40,8 +40,8 @@ class ActivityTotal(BaseModel):
     """
 
 
-class ActivityType(BaseModel):
-    __root__: Literal[
+class ActivityType(RootModel):
```

Comment:
All root models now inherit from the RootModel class  and use root: rather than __root__.

Reply:
Did you regenerate this model using the `datamodel_code_generator` tool (as we do in the bot)? If not, that's the very first thing we should do.